src/main/java/com/kenshoo/xlsx2csv/XlsxToCsvConverter.java

```diff
@@ -32,8 +31,11 @@
     private static final String COMMA = ",";
 
     private String delimiter = COMMA;
-    private CellHandler cellHandler = new CellHandler();
+    private int rowCacheSize = 10;
```

Comment:
change these to `private final `,  and initialize in default ctor from default values (constants).